Docs/source/running_cpp/parameters.rst

```diff
@@ -270,6 +270,12 @@ Particle initialization
     * ``NRandomPerCell``: injection with a fixed number of randomly-distributed particles per cell.
       This requires the additional parameter ``<species_name>.num_particles_per_cell``.
 
+    * ``singleparticle``: Inject a single macroparticle.
```

Comment:
Just for consistency
```suggestion
    * ``SingleParticle``: Inject a single macroparticle.
```
Note that the code converts it all to lower-case, but let's be consistent with what's written above.